src/marshmallow/fields.py

```diff
@@ -1112,15 +1106,26 @@ def _make_object_from_format(value, data_format):
         return dt.datetime.strptime(value, data_format)
 
 
-class LocalDateTime(DateTime):
-    """A formatted datetime string in localized time, relative to UTC.
+class NaiveDateTime(DateTime):
+    """A formatted naive datetime string."""
+    OBJ_TYPE = "naive datetime"
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        ret = super()._deserialize(value, attr, data, **kwargs)
+        if is_aware(ret):
+            raise self.fail("invalid", obj_type=self.OBJ_TYPE)
```

Comment:
Would there be value in using a different key in `default_error_messages` in order to allow users to have different messages for "invalid datetime" vs "invalid awareness"?

I've no strong opinion either way.

Reply:
Done.